pkg/apis/eksctl.io/v1alpha5/defaults.go

```diff
@@ -230,13 +230,22 @@ func getDefaultVolumeType(nodeGroupOnOutposts bool) string {
 	return DefaultNodeVolumeType
 }
 
-func setContainerRuntimeDefault(ng *NodeGroup) {
-	if ng.ContainerRuntime == nil {
-		ng.ContainerRuntime = &DefaultContainerRuntime
+func setContainerRuntimeDefault(ng *NodeGroup, apiVersion string) {
```

Comment:
`apiVersion` does not reflect that it refers to the cluster version. `clusterVersion` or something similar is a better name. 

```suggestion
func setContainerRuntimeDefault(ng *NodeGroup, clusterVersion string) {
```